packages/devtools_app/lib/src/inspector/flutter/inspector_tree_flutter.dart

```diff
@@ -147,6 +151,20 @@ class InspectorTreeControllerFlutter extends Object
     }
     return _maxIndent;
   }
+
+  bool get isDebugLayoutSummaryEnabled => _isDebugLayoutSummaryEnabled;
+  bool _isDebugLayoutSummaryEnabled = false;
+
+  void toggleDebugLayoutSummaryEnabled() {
+    _isDebugLayoutSummaryEnabled = !_isDebugLayoutSummaryEnabled;
+    if (animationController == null) return;
```

Comment:
why would the animationController be null?

Reply:
yeah i should remove this, i was thinking about the case when the inspector screen has not called initState